src/components/Nodes.js

```diff
@@ -16,19 +16,17 @@ export default class Nodes extends React.Component {
   }
 
   getUptimeData() {
-    axios.get('/api/nodes')
+    axios.get('http://localhost:3010/data')
```

Comment:
We have a special `DEV` mode you can run by starting `gulp` in one terminal window and `DEV=true node app.js` in the other. This mode creates a proxy to `browser-sync` server so you can use backend API without a need to care about CORS etc.

I also improved this mode to generate random measurements for each node. Please rebase master to use this new feature. Ping me in case of problems.